### ISSUE TITLE
Add pywinrm to the ansible-base virtual environment

### ISSUE THREAD

**Issue by mjneuharth** [actions/virtual-environments#3174]
**Tool information**
- Tool name: pywinrm
- Tool license:  MIT License (MIT license)
- Add or update? add
- Desired version: >= 0.4.1
- Approximate size: 
- If this is an add request:
  - Brief description of tool: Python library for Windows Remote Management
  - URL for tool's homepage: https://pypi.org/project/pywinrm/
  - Provide a basic test case to validate the tool's functionality: 
```
 import winrm

s = winrm.Session('windows-host.example.com', auth=('john.smith', 'secret'))
r = s.run_cmd('ipconfig', ['/all'])
>>> r.status_code
0
>>> r.std_out
Windows IP Configuration

   Host Name . . . . . . . . . . . . : WINDOWS-HOST
   Primary Dns Suffix  . . . . . . . :
   Node Type . . . . . . . . . . . . : Hybrid
   IP Routing Enabled. . . . . . . . : No
   WINS Proxy Enabled. . . . . . . . : No
...
>>> r.std_err
```

**Area for Triage**:
Python

**Question, Bug, or Feature?**:
Feature

**Virtual environments affected**
- [ ] Ubuntu 16.04
- [ ] Ubuntu 18.04
- [X] Ubuntu 20.04
- [ ] macOS 10.15
- [ ] macOS 11.0
- [ ] Windows Server 2016 R2
- [ ] Windows Server 2019

**Can this tool be installed during the build?**
pipx inject ansible-base "pywinrm>=0.4.1"

**Tool installation time in runtime**
~ 5seconds

**Are you willing to submit a PR?**
Sure.

**Comment by AlenaSviridenko** [comment]
Hi @mjneuharth,
given the fast installation time in runtime and installation simplicity, we'd rather not to include this package to the image due to maintenance concerns. 
I am closing this issue. Please, feel free to contact in case of any questions.

**Comment by mjneuharth** [comment]
I think that's a fair response. 

I raised it as an issue only because it was not obvious how to get the default Ansible (newly running in a pipx environment) to see pywinrm so that my pipeline job could connect to Windows hosts.

`pip install pywinrm` did not work and I had to dig through recent history on the [virtualenvironments](https://github.com/actions/virtual-environments) repo to understand what changed.

Would adding pywinrm without a specific version make the change more palatable?

Or is there another installation method (e.g. not injecting into base-ansible) that would be more maintainable and therefore acceptable?

Thanks for the quick response!
